src/content/learn/react-compiler/compatibility.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Backwards Compatibility
+---
+
+<Intro>
+React Compiler works best with React 19, but it also supports React 17 and 18 with additional configuration.
+</Intro>
+
+<YouWillLearn>
+
+* How to configure React Compiler for React 17 and 18
+
+</YouWillLearn>
+
+## Using React Compiler with React 17 or 18 {/*using-react-compiler-with-react-17-or-18*/}
+
+React Compiler works out of the box with React 19. For React 17 or 18, you need two things:
```

Comment:
❓ Question:
Could you please clarify the following:

> React Compiler works out of the box with React 19.

Does that mean that it's _enabled_ by default for anyone using React 19? (ie. if I use React 19 the compiler is running).
If so, can one disable it if they are using React 19?

"Works out of the box" can be interpreted in two ways:
- It works without extra setup, but you need to enable it
- It works without extra setup and it's enabled by default

The previous doc was saying:

> If you're starting a new project, you can enable the compiler on your entire codebase, which is the default behavior.

which was interpreted differently depending on who you ask

Thanks!

Reply:
Good question! Since the compiler is a separate package, "It works without extra setup, but you need to enable it" is what I meant. I can make this clearer